src/mainboard/starfive/visionfive2/bt0/src/main.rs

```diff
@@ -80,10 +69,11 @@ pub unsafe extern "C" fn start() -> ! {
         "csrw   mtvec, zero",
         // 1. suspend non-boot hart
         // hart 0 is the S7 monitor core; 1-4 are U7 cores
-        "li     a1, 0",
+        "li     a1, 1",
         "csrr   a0, mhartid",
         "bne    a0, a1, .nonboothart",
         // 2. prepare stack
+        // FIXME: each hart needs its own stack
```

Comment:
Since we are loading a payload, isn't the payload responsible for creating its runtime stacks?

Also, we are only running in the supervisor core with a `riscv64imac` target. All the other cores are running under the `riscv64gc` target (a superset of `riscv64imac`). I'm not sure about the safety of doing any setup that crosses that boundary. Thoughts?